modules/apps/list-type/list-type-service/src/test/java/com/liferay/list/type/service/ListTypeEntryLocalServiceTest.java

```diff
@@ -0,0 +1,230 @@
+		}
+	}
+
+	@Test
+	@TestInfo("LPD-55656")
+	public void testUpdateListTypeEntry() throws Exception {
+		try (SafeCloseable safeCloseable =
+				LazyReferencingThreadLocal.setEnabledWithSafeCloseable(true)) {
+
+			ListTypeEntry listTypeEntry =
+				_listTypeEntryLocalService.updateListTypeEntry(
+					RandomTestUtil.randomString(), RandomTestUtil.randomLong(),
+					RandomTestUtil.randomLocaleStringMap());
+
+			Assert.assertEquals(
+				WorkflowConstants.STATUS_APPROVED, listTypeEntry.getStatus());
+		}
+	}
+
+	private ListTypeDefinitionPersistence _mockListTypeDefinitionPersistence()
+		throws Exception {
+
+		ListTypeDefinition listTypeDefinition = Mockito.mock(
+			ListTypeDefinition.class);
+
+		Mockito.when(
+			listTypeDefinition.isSystem()
+		).thenReturn(
+			false
+		);
+
+		ListTypeDefinitionPersistence listTypeDefinitionPersistence =
+			Mockito.mock(ListTypeDefinitionPersistence.class);
+
+		Mockito.when(
+			listTypeDefinitionPersistence.findByPrimaryKey(Mockito.anyLong())
+		).thenReturn(
+			listTypeDefinition
+		);
+
+		return listTypeDefinitionPersistence;
+	}
+
+	private ListTypeEntryPersistence _mockListTypeEntryPersistence()
+		throws Exception {
+
+		ListTypeEntry listTypeEntry = Mockito.mock(ListTypeEntryImpl.class);
+
+		Mockito.doCallRealMethod(
+		).when(
+			listTypeEntry
+		).setStatus(
+			Mockito.anyInt()
+		);
+
+		listTypeEntry.setStatus(WorkflowConstants.STATUS_INCOMPLETE);
+
+		ListTypeEntryPersistence listTypeEntryPersistence = Mockito.mock(
+			ListTypeEntryPersistence.class);
+
+		Mockito.when(
+			listTypeEntryPersistence.findByPrimaryKey(Mockito.anyLong())
+		).thenReturn(
+			listTypeEntry
+		);
+
+		Mockito.when(
+			listTypeEntryPersistence.findByLTDI_K(
+				Mockito.anyLong(), Mockito.anyString())
+		).thenReturn(
+			null
+		);
+
+		Mockito.when(
+			listTypeEntryPersistence.create(Mockito.anyLong())
+		).thenReturn(
+			new ListTypeEntryImpl()
+		);
+
+		Mockito.when(
+			listTypeEntryPersistence.update(Mockito.any(ListTypeEntry.class))
+		).thenAnswer(
+			invocation -> invocation.getArgument(0)
+		);
+
+		return listTypeEntryPersistence;
+	}
+
+	private User _mockUser() {
+		User user = Mockito.mock(User.class);
+
+		Mockito.when(
+			user.getCompanyId()
+		).thenReturn(
+			RandomTestUtil.randomLong()
+		);
+
+		Mockito.when(
+			user.getUserId()
+		).thenReturn(
+			RandomTestUtil.randomLong()
+		);
+
+		Mockito.when(
+			user.getFullName()
+		).thenReturn(
+			RandomTestUtil.randomString()
+		);
+
+		return user;
+	}
+
+	private UserLocalService _mockUserLocalService(User user) throws Exception {
+		UserLocalService userLocalService = Mockito.mock(
+			UserLocalService.class);
+
+		Mockito.when(
+			userLocalService.getUser(Mockito.anyLong())
+		).thenReturn(
+			user
+		);
+
+		return userLocalService;
+	}
+
+	private final ListTypeDefinition _listTypeDefinition = Mockito.mock(
+		ListTypeDefinition.class);
+	private ListTypeEntryLocalService _listTypeEntryLocalService;
```

Comment:
@abesousa as far as I see this unit test class covers the same use cases already covered at ListTypeEntryLocalServiceTest by using integration tests, I saw some examples of unit tests in the portal for local service but the integration test seems to be easier to understand, I like unit tests and I believe we they will be mandatory soon, but for now to avoid situations with the second review step I would like you keep the integration tests and delete this one.

I'll save this unit test to encourage the team and Brian to accept more unit tests instead of always using integration tests as a silver bullet.

Thanks a lot for that!